features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,94 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
+    Given I store the nodes in the clipboard
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/ptp_master.yaml" replacing paths:
```

Comment:
@huiran0826 pls use the step `Given I obtain...` to get the data file

Reply:
@pruan-rht  Thanks, fixed it.